### PR TITLE
fix: support for multimethod 1.10

### PR DESCRIPTION
**Pull request by bopeng-saitama**
## Description

support for multimethod 1.10.

## Related links

Jira: https://tier4.atlassian.net/browse/RT2-984

## Pre-review checklist for the PR author

- [x] I've confirmed the [contribution guidelines](https://github.com/tier4/caret/blob/main/.github/CONTRIBUTING.md).
- [x] The PR is ready for review.

## In-review checklist for the PR reviewers

The PR reviewers **must** check the checkboxes below before approval.

- [x] The PR has been properly tested.
- [x] The PR has been reviewed.

## Post-review checklist for the PR author

The PR author **must** check the checkboxes below before merging.

- [ ] (Optional) The PR has been properly tested with CARET_report verification.
- [x] There are no open discussions or they are tracked via tickets.
- [x] The PR is ready for merge.

After all checkboxes are checked, anyone who has write access can merge the PR.

### CODE REVIEW

**Review by rokamu623 — CHANGES_REQUESTED** [review]
If not causing errors, the following is desirable.

File: src/caret_analyze/record/record_factory.py [hunk 1]
```diff
@@ -71,8 +71,8 @@ def _create_instance() -> RecordsInterface:
     @staticmethod
     @create_instance.register
     def _create_instance_record(
-        init: Sequence[RecordInterface] | None = None,
-        columns: Sequence[ColumnValue] | None = None
+        init: Sequence[RecordInterface],
+        columns: Sequence[ColumnValue]
```
[the remark this answers is not in the document]
Comment (bopeng-saitama) [reply src/caret_analyze/record/record_factory.py]: Fixed it in https://github.com/tier4/caret_analyze/pull/398/commits/875b94ef737a887dd788cd0ff59c2c86217e474d.

File: src/caret_analyze/record/records_service/response_time.py [hunk 2]
```diff
@@ -362,7 +362,7 @@ def to_worst_case_stacked_bar(self) -> RecordsInterface:
         # generate worst-case work flow
         filled_record_list: list[RecordInterface] = []
         for record_list in end_column_record_dict.values():
-            worst_record = RecordFactory.create_instance()
+            worst_record = RecordFactory.create_instance(None)
```
[the remark this answers is not in the document]
Comment (bopeng-saitama) [reply src/caret_analyze/record/records_service/response_time.py]: Fixed it in https://github.com/tier4/caret_analyze/pull/398/commits/875b94ef737a887dd788cd0ff59c2c86217e474d.

**Review by rokamu623 — APPROVED** [review]
LGTM!

**Review by xygyo77 — APPROVED** [review]
Trace data (https://github.com/tier4/CARET_report/releases/download/20221227/autoware_launch_trace_20221227-150253_universe_rosbag.zip ) to check the message flow graphs.
The following four graphs were png'd and the hash values (sha256sum) were confirmed to match.
 - mrm_emergency_stop_operator -> vehicle_cmd_gate_0 (6c3844da83fb9023c20c6615e4b3d1000000f26b3ce5d156ecc6ea17ea283728)  
 - mrm_emergency_stop_operator -> vehicle_cmd_gate_1   (f9cfc40d317289aa63bb90f3d1a9f7b1211682270838bf0e244d302539c55171)  
 - mrm_emergency_stop_operator -> vehicle_cmd_gate_2   (7ff851d6f0c9253bce9e9138c57a0e49912334b7f2e73835f0eb5a614d98e4e6)
 - behavior_path_planner -> obstacle_avoidance_planner_0   (7c8bc3c2c809b19e00923e270bd1e14f364646f25d06a50c3a9acfb344c3acd5)

**Pull request merged**